.rubocop.yml

```diff
@@ -79,6 +79,10 @@ Rails/FilePath:
 Rails/SaveBang:
   Enabled: true
 
+RescuedExceptionsVariableName:
+  Exclude:
+    - "spec/support/retry/pull_request_comment.rb"
```

Comment:
maybe we should disable this cop at all? otherwise it will always conflict with reek right?

or maybe we can accept `e` ? https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md

Reply:
We discussed this earlier today. If I name it "ex," RuboCop also asks me to change it to "e."
`Naming/RescuedExceptionsVariableName: Use 'e' instead of 'ex.'`

If I add to allow that 'e' is a valid variable name, it could potentially be used for anything, not just in this context. I believe this is the best option, and if it repeats, we may need to disable this cop.

Reply:
but why wouldn't you disable it at all? It will always cause problems, right? every time we catch an exception

Reply:
Because there was only one case so far, but yes is better to disable it to avoid future conflicts